pixi.toml

```diff
@@ -22,10 +22,10 @@ build = ["py311", "build"]
 lint = ["py311", "lint"]
 
 [dependencies]
+numba = "*"  # At top as this dictates Python version
```

Comment:
The order of the dependencies has some influence?

Reply:
This is because it would not solve on osx-64, see https://github.com/conda/rattler/issues/913.